pages/linux/lxc.md

```diff
@@ -0,0 +1,31 @@
+# lxc
+
+> Manages Linux containers (using the lxd REST API).
+
+- List all containers:
+
+`lxc list [{{remote}}:][{{filter}}]`
```

Comment:
This means you can use any one of:
`lxc list` to list all containers locally without a filter
`lxc list my-remote:` to list all containers on the `my-remote` private remote
`lxc list abc` to list all local containers matching the filter, `abc`
or
`lxc list my-remote:abc` to list all containers on the `my-remote` private remote matching `abc`

They must be in the order as written though as the value proceeding the colon is the remote.

Using the square brackets to denote optional parameters or arguments is pretty standard in Linux docs but I'm not sure how I should format this in tldr land. I'm also aware that tldr shouldn't have lengthy examples. Any advice gratefully received.